semeio/workflows/csv_export2/csv_export2.py

```diff
@@ -6,6 +6,70 @@
 from fmu import ensemble
 from res.job_queue import ErtScript
 
+DESCRIPTION = """
+CSV_EXPORT2 will export selected Eclipse summary vectors to a CSV file.
+The vector selection is independent of the ``SUMMARY`` keywords in the
+ert config file.
+
+The CSV file will look like:
+
+======== ==== =========== ==== ======
+ENSEMBLE REAL DATE        FOPR FOPT
+======== ==== =========== ==== ======
+iter-0   0    2020-01-01  800  0
+iter-0   0    2020-02-01  1000 365000
+iter-0   1    2020-01-01  700  0
+iter-0   1    2020-01-01  1100 401500
+======== ==== =========== ==== ======
+
+The time frequency must be chosen. If ``raw``, the original timesteps from
+Eclipse is chosen, and it will be individual pr. realization. If ``daily``,
+``weekly``, ``monthly``  or  ``yearly`` is chosen, only data at those dates are
+given for all realization. Rate data (e.g.  FOPR) is valid for the given dates,
+but can not be summed up to cumulative data when time interpolation. Cumulative
+columns (f.ex. FOPT) are time-interpolated linearly. See the `documentation on
+fmu-ensemble
+<https://equinor.github.io/fmu-ensemble/usage.html#rate-handling-in-eclipse-summary-vectors>`_
+for more details on rate handling.
+
+Columns are selected by a list of strings, where wildcards characters ``?``
+(matches exactly one character) and ``*`` (matches zero or more characters) can
+be used to select multiple columns.
+
+Column count more than 1000 gives increased probability for problems downstream,
+depending on which applications are put into use. Column count depends on the
+combination of wildcards used in this workflow and the actual vectors that are
+requested in the Eclipse DATA file. A wildcard like``W*`` could be dangerous on
```

Comment:
`dangerous`, at least to me, implies a security risk, which is not the case here, can you rephrase it to something else? For example `A wildcard like ``W*`` could lead to very large csv files on...`

Reply:
Agree. Fixed.